library-client-rtsp/src/main/java/com/alexvas/rtsp/widget/RtspSurfaceView.kt

```diff
@@ -57,7 +57,7 @@ open class RtspSurfaceView: SurfaceView {
     private val proxyClientListener = object: RtspClient.RtspClientListener {
 
         override fun onRtspConnecting() {
-            if (DEBUG) Log.v(TAG, "onRtspConnecting()")
+            if (DEBUG) Log.v(TAG, "onRtspConnecting..")
```

Comment:
if (DEBUG) Log.v(TAG, "functionName()")
Should not be touched.

Reply:
We did that for testing purpose , will revert it back.